src/math-editor.js

```diff
@@ -98,6 +107,12 @@ export function init(
             $latexField.val(latex)
             updateMathImgWithDebounce($mathEditorContainer.prev(), latex)
             updateLatexFieldHeight()
+            if (!undoing && undoablesStack.last() !== latex) {
```

Comment:
I would prefere not extend Array prototype. Maybe just call `last(undoablesStack)`?